docs/FHIRServerUsersGuide.md

```diff
@@ -1167,87 +1167,6 @@ and then we could modify the user-defined schematron rule as follows:
 
 By doing this, we have a much more flexible user-defined validation mechanism that allows us to apply rules only when 'http://ibm.com/fhir/profile/partner' is present in the instance.
 
-## 4.5 Encryption and decryption of requests and responses
```

Comment:
The sucky thing about our current markdown format is that when you remove a section, you need to update the section numbers of all the sections that follow it.  Sorry :-(

Reply:
And the index at the top of the document :-/